### PR TITLE
[Tests-Only] move appsTest to jest

### PR DESCRIPTION
**Pull request by dpakach**
move appsTest to jest and pactV3

part of #645 & #646

### CODE REVIEW

File: tests/capabilitiesTest.js [hunk 1]
```diff
@@ -1,8 +1,6 @@
 import { MatchersV3, XmlBuilder } from '@pact-foundation/pact/v3'
 
 describe('Main: Currently testing getConfig, getVersion and getCapabilities', () => {
-  let oc
```
Comment (individual-it) [inline tests/capabilitiesTest.js:4]: good changes, but should be in a different PR

File: tests/appsTest.js [hunk 2]
```diff
@@ -1,229 +1,168 @@
...
+      })
   }
 
-  beforeAll(function () {
-    const promises = []
-    promises.push(provider.addInteraction(CORSPreflightRequest()))
-    promises.push(provider.addInteraction(capabilitiesGETRequestValidAuth()))
-    promises.push(provider.addInteraction(GETRequestToCloudUserEndpoint()))
-    // a request to GET attributes of an app
-    const attributes = {
```
Comment (individual-it) [inline tests/appsTest.js:39]: please make a note in the PR about why all these interactions are not needed

**Review by individual-it — CHANGES_REQUESTED** [review]

File: tests/appsTest.js (hunk 2 above)
Comment (dpakach) [reply tests/appsTest.js:39]: Tests using these interactions were removed in https://github.com/owncloud/owncloud-sdk/pull/615

**Review by individual-it — APPROVED** [review]
